package.json

```diff
@@ -6,8 +6,13 @@
   "scripts": {
     "drop-users-table": "babel-node ./server/data/relations dropUsersTable",
     "create-users-table": "babel-node ./server/data/relations createUsersTable",
+    "drop-parties-table": "babel-node ./server/data/relations dropPartiesTable",
+    "create-parties-table": "babel-node ./server/data/relations createPartiesTable",
+    "drop-tables": "npm run drop-users-table && npm run drop-parties-table",
+    "create-tables":"npm run create-users-table && npm run create-parties-table",
```

Comment:
Would you mind having one script to create all tables and one script to drop them??
but also make sure that both scripts are being run before the test script

Reply:
Sure! Going to work on it.